extensions/wikia/Recirculation/js/helpers/LiftigniterHelper.js

```diff
@@ -51,13 +51,18 @@ define('ext.wikia.recirculation.helpers.liftigniter', [
 			}
 
 			$.each(data.items, function (index, item) {
-				item.isWiki = item.source === 'wiki';
+				item.isWiki = false;
 
 				if (items.length < options.max && item.thumbnail) {
 					item.source = options.source;
 					item.meta = options.widget;
 					item.index = index;
 
+                    if (item.source === 'wiki') {
+                        item.isWiki = true;
+                        item.thumbnail = thumbnailer.getThumbURL(item.thumbnail, 'image', options.width, options.height);
```

Comment:
I'm piggybacking in this change because when making the slides I noticed that the images listed in the spreadsheet don't fit the right aspect ratio. Rather than having comdev go through and recreate those images I'll just pass them through Vignette